### PR TITLE
shell: Avoid spurious commas in ssh-keyscan command example

### PR DESCRIPTION
**Pull request by mvollmer**
Passing more than one child to ClipboardCopy seems to introduce them, let's just pass a single string.

### CODE REVIEW

File: pkg/shell/hosts_dialog.jsx [hunk 1]
```diff
@@ -474,7 +476,7 @@ class HostKey extends React.Component {
                 <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
                 <p className="hostkey-type">({key_type})</p>
                 <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
+                       <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">{scan_cmd}</ClipboardCopy>
```
Comment (cockpituous) [inline pkg/shell/hosts_dialog.jsx:479]: This added line is not executed by any test.

**Review by martinpitt — APPROVED** [review]
That's a strong déjà vu, I take it it's from a bigger PR that I already reviewed but got stuck? Thanks!

File: pkg/shell/hosts_dialog.jsx [hunk 2]
```diff
@@ -474,7 +476,7 @@ class HostKey extends React.Component {
                 <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
                 <p className="hostkey-type">({key_type})</p>
                 <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
+                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">{scan_cmd}</ClipboardCopy>
```
Comment (cockpituous) [inline pkg/shell/hosts_dialog.jsx:479]: This added line is not executed by any test.

**Comment by mvollmer** [comment]
> That's a strong déjà vu, I take it it's from a bigger PR that I already reviewed but got stuck?

We fixed this in the pkg/lib/ version of these dialogs already, and I made a note to fix it in the pkg/shell version as well. Took a bit to get to it... :)

**Pull request merged**